home/views.py

```diff
@@ -43,3 +43,37 @@ def contact(request):
 def logoutUser(request):
     logout(request)
     return redirect("login")
+
+@login_required
+def categories(request):
+    if request.method == 'POST':
+        form = CategoryForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return redirect('categories') 
+    else:
```

Comment:
Else is not necessary

Reply:
Sir, should I delete "else:..." in both categories and products functions?